.cirrus.yml

```diff
@@ -58,13 +76,31 @@ task:
   <<: *standard_build
 
 task:
-  name: FreeBSD 12.0 amd64
-  gce_instance:
-    <<: *standard_gce_instance
-    image_project: freebsd-org-cloud-dev
-    image_name: freebsd-12-0-release-amd64
-    disk: 30  # Image is 29GB, VM can't be smaller
```

Comment:
`disk` can't be dropped as the FreeBSD image has a larger disk. I guess we could make it larger in the `standard_gce_instance` but otherwise we need to include this on all FreeBSD instances.